runtime/src/iree/tooling/buffer_view_matchers.c

```diff
@@ -76,14 +76,21 @@ static bool iree_hal_compare_strided_elements_exact(
   return true;
 }
 
-static bool iree_hal_compare_strided_elements_approximate_absolute_f16(
+static bool iree_hal_compare_strided_elements_approximate_f16(
```

Comment:
it's better to keep unique functions as before - add a new _relative_ version - adding bool args makes the code harder to follow and less efficient (you want the conditions on the outside of the loop, not the inside - this may be evaluated millions of times)